client/tests/integration/smartcontracts/validator_with_custom_token/src/lib.rs

```diff
@@ -299,32 +191,3 @@ pub fn migrate(_block_height: u64) -> MigrationResult {
 
     Validator::replace_token(&accounts)
 }
-
-#[entrypoint]
-pub fn validate_transaction(
-    authority: AccountId,
-    transaction: VersionedSignedTransaction,
-    block_height: u64,
-) -> Result {
-    let mut validator = Validator::new(block_height);
-    validator.visit_transaction(&authority, &transaction);
-    validator.verdict
-}
-
-#[entrypoint]
-pub fn validate_instruction(
-    authority: AccountId,
-    instruction: InstructionBox,
-    block_height: u64,
-) -> Result {
-    let mut validator = Validator::new(block_height);
-    validator.visit_instruction(&authority, &instruction);
-    validator.verdict
-}
-
-#[entrypoint]
-pub fn validate_query(authority: AccountId, query: QueryBox, block_height: u64) -> Result {
-    let mut validator = Validator::new(block_height);
-    validator.visit_query(&authority, &query);
-    validator.verdict
-}
```

Comment:
To be honest, I'd prefer to leave all `entrypoint`s as is. So that it will be clear which functions can be executed by the host

Reply:
it doesn't sound entirely logical to me that this is part of the `default_validator` macro

we can:
1. leave it as is (user will copy this bloat for every validator)
2. make it part of the `default_validator` as was done in this PR. (But we need a way to be able to provide custom impls)
3. make another macro that is somehow related to `default_validator` macro (it needs to know the name of the validator struct generated by it)

Reply:
I think keeping the entrypoints separate is a good idea. Bikeshed syntax:

```rust
default_entrypoints!{
  [
    validate_transaction,
    validate_instruction,
    validate_query,
  ] => MyValidator
}
```

Reply:
Went for a `#[omit(entrypoints(..)` syntax here, should work just as well.